### ISSUE TITLE
Following the Wiki for persisting mount does not work

### ISSUE THREAD

**Issue by misterjaytee** [Azure/azure-storage-fuse#53]
You can use blobfuse without installing fuse, however...

If you follow the Wiki for Configuring and Running blobfuse, the method for persisting the mount by adding an entry to /etc/fstab does not work unless you have already installed fuse.

You will get an error, like the one below, when you run "mount -a":
```
mount: wrong fs type, bad option, bad superblock on /scripts/mount.sh,
       missing codepage or helper program, or other error

       In some cases useful info is found in syslog - try
       dmesg | tail or so.
```

Looking at dmesg doesn't help.

For Red Hat systems (such as Centos, fedora etc.):
yum install fuse

It took me a while to work this out because blobfuse was working fine with the manual mount. So putting this here for the next poor soul that comes across this problem (or for someone to put into the Wiki).

**Comment by misterjaytee** [comment]
As we don't seem to be able to do pull requests for Git Wikis, I have cloned the Wiki and made appropriate updates which you can use if you wish:
https://github.com/misterjaytee/azure-storage-fuse-wiki

**Comment by seguler** [comment]
Thanks for reporting this! I don't quite understand exactly though - how can you mount blobfuse while you don't have fuse module ? And I haven't ever seen a distro where fuse wasn't installed. What version of CentOs did you have ?

Also the wiki included a command to install fuse as a per-requisite already. You wouldn't be able to compile a binary if you didn't have fuse installed... Could it be that something else resolved your problem ?

**Comment by misterjaytee** [comment]
Okay, I see why you're confused - you assumed I built from source...

The machine is a Virtual Machine from an Azure image (OpenLogic Centos 7.4). The install instructions for installing from a Repository do not mentioned installing fuse. I'd guess that if this is a dependency (or specific version is a dependency) then it should get installed at the same time when installing from a Repo.

Because this is an Azure VM, the Microsoft repo was already setup, so `yum install blobfuse` was all that was required. Following the instructions, it says "You're set to go! Now follow the Configuring and Running wiki...", so I did - which meant I didn't read through the Build Instructions as I wasn't building from scratch (if I had read through them, I'd probably have noticed the dependency list.

Maybe what is required is for the rpm installer to make sure fuse is installed.
